packages/components/organisms/f-error-page/src/components/ErrorPage.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+    <div
+        :class="$style['c-errorPage']"
+        data-test-id="errorPage">
+        {{ copy.text }}
+    </div>
+</template>
+
+<script>
+import { globalisationServices } from '@justeat/f-services';
+import tenantConfigs from '../tenants';
+/* eslint-disable */
+//import ErrorPageServiceApi from '../services/ErrorPageServiceApi';
```

Comment:
Is this needed at this stage?

Reply:
Will probably be deleted in the next PR. Commented out for now as it was causing a eslint error.